tests/run-tests.js

```diff
@@ -40,12 +36,12 @@ const results = testDirs.map((test) => {
     };
   }
 
-  if (actualOutput.toString() !== expectedOutput.toString()) {
+  if (actualOutput !== expectedOutput) {
```

Comment:
I changed the type of these variables from buffers to strings to call toString() a bit less often.